rllib/agents/ddpg/apex.py

```diff
@@ -17,41 +20,49 @@
         "n_step": 3,
         "num_gpus": 0,
         "num_workers": 32,
-        "buffer_size": 2000000,
-        # TODO(jungong) : update once Apex supports replay_buffer_config.
-        "no_local_replay_buffer": True,
-        # Whether all shards of the replay buffer must be co-located
-        # with the learner process (running the execution plan).
-        # This is preferred b/c the learner process should have quick
-        # access to the data from the buffer shards, avoiding network
-        # traffic each time samples from the buffer(s) are drawn.
-        # Set this to False for relaxing this constraint and allowing
-        # replay shards to be created on node(s) other than the one
-        # on which the learner is located.
-        "replay_buffer_shards_colocated_with_driver": True,
-        "learning_starts": 50000,
+        "replay_buffer_config": {
+            "capcity": 2000000,
+            "no_local_replay_buffer": True,
+            # Specify prioritized replay by supplying a buffer type that supports
+            # prioritization
+            "prioritized_replay": DEPRECATED_VALUE,
```

Comment:
With your new config, is the `prioritized_replay` option still required?

Reply:
No. I just wanted to leave it there since it's being deprecated!